lib/screen/note_screen.dart

```diff
@@ -296,7 +292,7 @@ class _NoteScreen extends State<NoteScreen> with CommandHandler {
   /// Callback on a single note clicked
   void _onNoteTap(Note note) async {
     final command =
-    await Navigator.pushNamed(context, '/note', arguments: {'note': note});
+    await Navigator.pushNamed(context, '/note_editor', arguments: {'note': note});
```

Comment:
So we're basically navigating to the node editor when the note is clicked?

Reply:
Yes exactly, I just rename the route to note_editor. The route name we had there was note but redirect to the editor, so I changed to an explicit name